prov/verbs/src/verbs_info.c

```diff
@@ -243,9 +243,6 @@ static int vrb_check_hints(uint32_t version, const struct fi_info *hints,
 		return -FI_ENODATA;
 	}
 
-	if (!ofi_valid_addr_format(info->addr_format, hints->addr_format))
-		return -FI_ENODATA;
-
```

Comment:
why remove this check?

Reply:
This check is redundant since we moved it to vrb_filter_info_by_addr_format().

Reply:
It's not redundant. User may pass an invalid addr_format. It should be check here instead of taking all the effort to set it and then remove it later.

Reply:
Corrected in recent push.